tasmota/language/en_GB.h

```diff
@@ -1270,4 +1271,10 @@
 #define D_SENSOR_PIPSOLAR_TX             "Pipsolar TX"
 #define D_SENSOR_PIPSOLAR_RX             "Pipsolar RX"
 
+// xsns_115_wooliis.ino
+#define D_CHARGING                        "Charging"
+#define D_INGRESS                         "Ingress"
+#define D_EGRESS                          "Egress"
+#define D_CAPACITY                        "Capacity"
```

Comment:
Pls add this to all language files. Doesn't matter it is english as they will be translated eventually.